src/content/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/alerts-ai-transition-guide-2022.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Alerts and AI transition guide 2022
```

Comment:
We aren't supposed to use 'AI' as an acronym, and are only supposed to say 'applied intelligence'. 

Also, weirdly, we apparently no longer capitalize it and it's meant to be sentence case now (e.g., 'applied intelligence'). I find this strange, but it's apparently what was decided (let me know if you want that link to marketing guidelines). And also we don't currently use sentence case for this and other AI-related terms in the docs, so that's another weirdness that maybe means we just leave it as title case treatment for now.

Reply:
good to know, thanks!